src/components/organisms/Notifications.tsx

```diff
@@ -0,0 +1,264 @@
+    is_read: false,
+  },
+  {
+    id: 8,
+    date: "15/05/22 11:20",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 9,
+    date: "01/07/22 15:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 10,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 11,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 12,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 13,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 14,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 15,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 16,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 17,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 18,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 19,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 20,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 21,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 22,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 23,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 24,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 25,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 26,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 27,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 28,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+];
+
+// ToDo -> implement when message click
+const handleClickMessage = (data: any) => {};
+
+export default function Notifications() {
+  const [readState, setReadState] = useState<string>("all");
+  const [notifiData, setNotifiData] = useState<Array<NotificationsData> | undefined>([]);
+
```

Comment:
Change the type like below. I delete the `undefined` because it will not be set to the `notifiData`.
```ts
useState<NotificationsData[]>([]);
```